### ISSUE TITLE
A11y Screen Reader - Page structure challenges

### ISSUE THREAD

**Issue by DaveBest99** [angular/material.angular.io#111]
Screen reader WCAG issues.
1. Perceivable: Page structure lacks content relationships; (no <main> region), (missing text label on <nav>), (missing text label on <footer>).
2. Operable: Pressing Enter key on Link elements (like "COMPONENTS" and "GUIDES") provide no verbal feedback of cursor focus change to notify the blind user.
3. Understandable: The page is simple but is missing Breadcrumb and proper page Title for orientation.
4. Robust: Not tested.
5. See attached files.

[Angular Accessibility Introduction.docx](https://github.com/angular/material.angular.io/files/800324/Angular.Accessibility.Introduction.docx)

Angular Material Accessibility Introduction.mp4: https://www.dropbox.com/s/rz2gtuhnbx9czus/Angular%20Material%20Accessibility%20Introduction.mp4?dl=0

**Comment by naomiblack** [comment]
@riavalon could you update this issue with status? Other than the issue with routing, where are we at on the remainder of these page structure questions?

**Comment by riavalon** [comment]
Sure thing: I've added in some tweaks to update how the page is structured with different regions as well as added the ability to have the page title updated on route change now. There is still an issue with getting verbal feedback about page transitions that I don't think we have officially settled on a solution for. There's more info in the PR I created: #117
